guides/main_concepts/documents.md

```diff
@@ -1,68 +1,90 @@
+An attribute is like a variable or a property in most programming languages: a name that allows you to store and access some data. That data is what we call the attribute's **value**.
 
-**MeiliSearch uses a lot of RAM when indexing documents**, be aware of your RAM availability as you increase the size of your batch as this could result in a MeiliSearch crash.
+Every field has a [data type](/guides/advanced_guides/datatypes.md) dictated by its value. Every value must be a valid [`JSON` data type](https://www.w3schools.com/js/js_json_datatypes.asp).
 
-## Fields
+Take note that in the case of string values, the string **[must contain 1000 words or fewer](/guides/advanced_guides/known_limitations.html#maximum-words-per-attribute)**. If it contains more than 1000 words, only the first 1000 will be indexed.
+
+You can also apply [<clientGlossary word="ranking rules" />](/guides/main_concepts/relevancy.md#ranking-rules) to some fields. For example, you may decide recent movies should be more relevant than older ones.
+
+If you would like to adjust how a field gets handled by MeiliSearch, you can do so in the [settings](/guides/advanced_guides/settings.md#settings).
 
-- **[Data type](/guides/advanced_guides/datatypes.md)**: A field's data type determines what kind of data can be stored in that field.
-- **[Field properties](/guides/advanced_guides/field_properties.md)**: Field properties determines the characteristics and behavior of the data added to that field.
+### Field properties
 
-By default, all fields included in a document are <clientGlossary word="searchable" /> and <clientGlossary word="displayed" />.
-You can adjust how a field get handled by MeiliSearch in the settings. It can either be searchable or displayed, both, or none of both. In the latter case, the field will be completely ignored when a document is sent.
+A field may also possess **[field properties](/guides/advanced_guides/field_properties.md)**. Field properties determine the characteristics and behavior of the data added to that field.
 
-You can also apply <clientGlossary word="ranking rules" /> to some fields. For example, you may decide recent movies should be more relevant than older ones.
+At this time, there are two field properties: [<clientGlossary word="searchable" />](/guides/advanced_guides/field_properties.md#searchable-fields) and [<clientGlossary word="displayed" />](/guides/advanced_guides/field_properties.md#displayed-fields). A field can have one, both, or neither of these properties. **By default, all fields in a document are both displayed and searchable.**
+
+To clarify, a field may be:
+* searchable but not displayed
+* displayed but not searchable
+* both displayed and searchable (default)
+* neither displayed nor searchable
+
+In the latter case, the field will be completely ignored when a search is performed. However, it will still be [stored](/guides/advanced_guides/field_properties.md#data-storing) in the document.
```

Comment:
I think "attributeForFaceting" is also a field property. But contrary to the two other, attributeForFaceting is by default empty. 
Maybe we should simplify this part and add information about attributeForFaceting.
What do you think ?